includes/compat/class-wc-stripe-subscriptions-helper.php

```diff
@@ -215,6 +211,20 @@ public static function is_manual_renewal_enabled() {
 		return false;
 	}
 
+	/**
+	 * Extracts data from a subscription object for detached subscriptions.
+	 *
+	 * @param $subscription WC_Subscription The subscription object to extract data from.
+	 * @return array
+	 */
+	public static function get_detached_data_from_subscription( $subscription ) {
```

Comment:
Optional suggestion,
```suggestion
	public static function get_detached_payment_data_from_subscription( $subscription ) {
```